src/main/java/com/mixpanel/android/viewcrawler/ViewCrawler.java

```diff
@@ -175,18 +175,23 @@ public void onActivityCreated(Activity activity, Bundle bundle) {
 
         @Override
         public void onActivityStarted(Activity activity) {
+            ConnectivityIndicator.start(mContext);
+            mCurrentActivity = activity;
         }
 
         @Override
         public void onActivityResumed(Activity activity) {
             installConnectionSensor(activity);
             mEditState.add(activity);
+            mCurrentActivity = activity;
+            ConnectivityIndicator.show(activity);
```

Comment:
it is better to have the check inside show as then in future when people reuse the method it won't crash or cause issues in the app. Because this check needs to be done everytime show is issued